Update0304

christianneurohr opened the pull request:
See commit

Commit:
Merge branch 'Changes1303' of github.com:christianneurohr/hcperiods

Commit:
Merge branch 'master' of https://github.com/pascalmolin/hcperiods

Commit:
- Typos

Commit:
Merge branch 'master' of https://github.com/pascalmolin/hcperiods

Commit:
- Wrote paragraph on spanning tree
- Some minor corrections

pascalmolin commented:
OK (note that mentionning D in the spanning tree complexity is irrelevant since D is fixed, this can be removed)

Commit:
- Rewrote spanning tree paragraph, espc. removed D
- Started working on Intersections again
- Joined two images into one, saving space

Commit:
- Almost finished section on intersections

Commit:
- Finished section on Intersections (I think its pretty solid now)
- Deleted unnecessary tikz.files
